Nfield.Quota/Builders/QuotaFrameBuilder.cs

```diff
@@ -49,23 +49,26 @@ public QuotaFrameBuilder Target(int? target)
         public QuotaFrameBuilder VariableDefinition(
             string variableName,
             string odinVariableName,
-            IEnumerable<string> levelNames)
+            IEnumerable<string> levelNames,
+            bool? isSelectionOptional = null)
```

Comment:
Rather than a nullable bool, could we use an enum instead?

Reply:
We'll do it